pykickstart/commands/sshkey.py

```diff
@@ -58,6 +58,17 @@ def _getArgsAsStr(self):
         retval += ' "%s"' % self.key
         return retval
 
+class F38_SshKeyData(F22_SshKeyData):
+    removedKeywords = BaseData.removedKeywords
+    removedAttrs = BaseData.removedAttrs
+
+    def _getArgsAsStr(self):
+        retval = ""
+
+        retval += " --username=%s" % self.username
+        retval += ' "%s"' % self.key.replace('"', r'\"')
```

Comment:
I wonder if it would make sense to use `shlex.quote` - at a glance, that's the exact situation it was made for?

Reply:
You would think so, wouldn't you :) But that's not actually what it does. shlex can unescape `\"` but it doesn't generate them.